src/main/java/com/radixdlt/crypto/ECDSASignatures.java

```diff
@@ -1,30 +1,94 @@
 package com.radixdlt.crypto;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import com.radixdlt.serialization.DsonOutput;
+import com.radixdlt.serialization.SerializerConstants;
+import com.radixdlt.serialization.SerializerDummy;
 import com.radixdlt.serialization.SerializerId2;
 
 import java.util.Map;
 
 @SerializerId2("crypto.ecdsa_signatures")
-public class ECDSASignatures extends SignaturesImpl<ECDSASignature> {
+public class ECDSASignatures implements Signatures {
+    // Placeholder for the serializer ID
+    @JsonProperty(SerializerConstants.SERIALIZER_NAME)
+    @DsonOutput(DsonOutput.Output.ALL)
+    private SerializerDummy serializer = SerializerDummy.DUMMY;
+
+    @JsonProperty("version")
+    @DsonOutput(DsonOutput.Output.ALL)
+	private short version = 100;
```

Comment:
is Github drunk? or is there some whitespace before variable `version`?

Reply:
Yes, github does tabs in a completely messed up way.  Tabs are 8 spaces by default, but the counting of the 8 spaces starts to the left of the left margin, so the first tab ends up being six spaces wide.

Looks weird here, because @JsonProperty and @DsonOutput are indented using 4 spaces, and the field declaration is indented with a tab, which ends up being 6 chars wide here.

@flotothemoon , can you help @Sajjon get his IDE configured to use tabs instead of spaces?

Reply:
I have it setup now!